lib/bunny/session.rb

```diff
@@ -337,14 +337,14 @@ def transport_write_timeout
     # opened (this operation is very fast and inexpensive).
     #
     # @return [Bunny::Channel] Newly opened channel
-    def create_channel(n = nil, consumer_pool_size = 1, consumer_pool_abort_on_exception = false)
+    def create_channel(n = nil, consumer_pool_size = 1, consumer_pool_abort_on_exception = false, consumer_pool_shutdown_timeout = nil)
```

Comment:
Again, a non-nil timeout (maybe up to 60 seconds?) sounds OK and would make most people benefit from this improvement.